### PR TITLE
Replaces PNG host endpoint diagram with SVG

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Adds an SVG copy of the host endpoint diagram in bare-metal.md
- Changes bare-metal.md to reference the SVG copy.
- Tested this locally, renders well.
- Allows us to more easily maintain this diagram and make changes, also improves visual appearance of the diagram with vector image

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: images/bare-metal-packet-flows.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0.203125 0.203125 0.3125 0.484375q0.125 0.28125 0.125 0.59375q0 0.3125 -0.125 0.59375q-0.109375 0.265625 -0.3125 0.46875q-0.203125 0.203125 -0.484375 0.3125q-0.28125 0.125 -0.578125 0.125q-0.328125 0 -0.609375 -0.125q-0.265625 -0.109375 -0.46875 -0.3125q-0.203125 -0.203125 -0.328125 -0.46875q-0.109375 -0.28125 -0.109375 -0.59375q0 -0.3125 0.109375 -0.59375q0.125 -0.28125 0.328125 -0.484375q0.203125 -0.203125 0.46875 -0.328125q0.28125 -0.125 0.609375 -0.125zm14.647919 -3.078125q0 1.375 -0.28125 2.5q-0.265625 1.125 -0.828125 1.9375q-0.546875 0.796875 -1.375 1.234375q-0.8125 0.4375 -1.9375 0.4375q-0.953125 0 -1.734375 -0.359375q-0.78125 -0.359375 -1.34375 -1.09375q-0.546875 -0.75 -0.859375 -1.90625q-0.296875 -1.15625 -0.296875 -2.75q0 -1.359375 0.28125 -2.484375q0.28125 -1.140625 0.828125 -1.9375q0.546875 -0.8125 1.375 -1.25q0.84375 -0.4375 1.9375 -0.4375q0.953125 0 1.734375 0.359375q0.796875 0.359375 1.34375 1.109375q0.5625 0.75 0.859375 1.90625q0.296875 1.15625 0.296875 2.734375zm-1.59375 0.0625q0 -0.3125 -0.03125 -0.609375q-0.015625 -0.3125 -0.046875 -0.609375l-5.046875 3.75q0.140625 0.46875 0.34375 0.875q0.21875 0.390625 0.515625 0.6875q0.296875 0.28125 0.671875 0.453125q0.375 0.1
```
Comment (fasaxc) [inline images/bare-metal-packet-flows.svg]: Looks like the old diagram; I think @song-jiang is sending over the new version

Do SVGs render well in all current browsers?

Should you remove the old image? (Or is it still in use for the older versions of the site?)

Comment (emanic) [reply images/bare-metal-packet-flows.svg]: OK, I'll look for that. Thanks! Yes, the SVG renders well and it's well supported now. I kept the old PNG in the older version directories for expediency.

**Comment by tomdee** [comment]
@emanic Is this PR still in progress?

**Comment by emanic** [comment]
Sorry for the delay on this one. I've updated it now with the correct SVG. Can you take another look @fasaxc ? Renders well.

**Pull request merged**